specification/deviceprovisioningservices/resource-manager/readme.md

```diff
@@ -105,6 +114,19 @@ swagger-to-sdk:
       - node sdkauto_afterscript.js deviceprovisioningservices/resource-manager
 ```
 
+## Suppression
+``` yaml
+directive:
+  - suppress: SECRET_PROPERTY
+    from: 
+      - Microsoft.Devices/preview/2021-02-01-preview/iotdps.json
+    where: 
+      - $.definitions.SharedAccessSignatureAuthorizationRule[AccessRightsDescription].properties.primaryKey
```

Comment:
Square bracket is not supported in property name when do suppression.

Reply:
Will leave the bracket in property name for now and will depend on a force merge for the validation failures caused by this.
Long term fix will be false alarms mentioned [here](https://github.com/Azure/azure-rest-api-specs/pull/12378#issuecomment-761957826)